src/API/public/speciesList.json

```diff
@@ -0,0 +1,139 @@
+    {"species":"deemingi"},
+    {"species":"demeilloni"},
+    {"species":"distinctus"},
+    {"species":"domicola"},
+    {"species":"dthali"},
+    {"species":"dualaensis"},
+    {"species":"dureni"},
+    {"species":"eouzani"},
+    {"species":"erepens"},
+    {"species":"erythraeus"},
+    {"species":"ethiopicus"},
+    {"species":"faini"},
+    {"species":"flavicosta"},
+    {"species":"fontenillei"},
+    {"species":"fontinalis"},
+    {"species":"freetownensis"},
+    {"species":"fuscicolor"},
+    {"species":"fuscivenosus"},
+    {"species":"gabonensis"},
+    {"species":"garnhami"},
+    {"species":"gibbinsi"},
+    {"species":"grassei"},
+    {"species":"grenieri"},
+    {"species":"griveaudi"},
+    {"species":"hamoni"},
+    {"species":"hancocki"},
+    {"species":"hargreavesi"},
+    {"species":"harperi"},
+    {"species":"hervyi"},
+    {"species":"hughi"},
+    {"species":"implexus"},
+    {"species":"jebudensis"},
+    {"species":"keniensis"},
+    {"species":"kingi"},
+    {"species":"kosiensis"},
+    {"species":"lacani"},
+    {"species":"leesoni"},
+    {"species":"letabensis"},
+    {"species":"listeri"},
+    {"species":"lloreti"},
+    {"species":"longipalpis"},
+    {"species":"lounibosi"},
+    {"species":"lovettae"},
+    {"species":"machardyi"},
+    {"species":"maculipalpis"},
+    {"species":"maliensis"},
+    {"species":"marshallii"},
+    {"species":"mascarensis"},
+    {"species":"millecampsi"},
+    {"species":"milloti"},
+    {"species":"mortiauxi"},
+    {"species":"moucheti"},
+    {"species":"mousinhoi"},
+    {"species":"multicolor"},
+    {"species":"multicinctus"},
+    {"species":"murphyi"},
+    {"species":"namibiensis"},
+    {"species":"natalensis"},
+    {"species":"njombiensis"},
+    {"species":"notleyi"},
+    {"species":"obscurus"},
+    {"species":"okuensis"},
+    {"species":"ovengensis"},
+    {"species":"paludis"},
+    {"species":"parensis"},
+    {"species":"pauliani"},
+    {"species":"pharoensis"},
+    {"species":"pretoriensis"},
+    {"species":"radama"},
+    {"species":"rageaui"},
+    {"species":"ranci"},
+    {"species":"rhodesiensis"},
+    {"species":"rivulorum"},
+    {"species":"rivulorum-like"},
+    {"species":"rodhaini"},
+    {"species":"roubaudi"},
+    {"species":"ruarinus"},
+    {"species":"rufipes"},
+    {"species":"salbaii"},
+    {"species":"schwetzi"},
+    {"species":"seretsei"},
+    {"species":"sergentii"},
+    {"species":"seydeli"},
+    {"species":"smithii"},
+    {"species":"somalicus"},
+    {"species":"squamosus"},
+    {"species":"stephensi"},
+    {"species":"swahilicus"},
+    {"species":"symesi"},
+    {"species":"tchekedii"},
+    {"species":"tenebrosus"},
+    {"species":"theileri"},
+    {"species":"turkhudi"},
+    {"species":"vaneedeni"},
+    {"species":"vanhoofi"},
+    {"species":"vernus"},
+    {"species":"vinckei"},
+    {"species":"walravensi"},
+    {"species":"wellcomei"},
+    {"species":"wilsoni"},
+    {"species":"ziemanni"}
+  ]
+}
```

Comment:
Well done typing this all out, but... Can't we just get this list from the db? Then we won't have to maintain two sources of truth for the species list.

Reply:
I'm unsure of the best way to approach this. When either yourself or Ray returns we can have a brief call to fill me in